Sign In, Sign Up, and Logout

piseilim opened the pull request:
Please take a look and let me know if there's any merge conflict.

o2badad committed:
Postgresql was added to the gemfile and configured for our database server.
PG assumes local use has db create permissions with admin access.
PG assumes no password for local user.

Ruby version 2.2.2 added to Gemfile as well.

o2badad committed:
Merge branch 'master' of https://github.com/MurrietaRoR/shopping-cart into MurrietaRoR/master

o2badad committed:
Merge branch 'master' of https://github.com/MurrietaRoR/shopping-cart into MurrietaRoR/master
There were conflicts in the Gemfile I cleaned up. They were just comments, so no critical
changes were made.

Commit:
Merge branch 'chan/front-end'

o2badad committed:
Product, Category, Order and Order_Item models are setup for basic function.
Seed file is setup with some data to display. Make sure to db:migrate and db:setup.

Commit:
Added signup, login, and logout

o2badad committed:
Updated pages home view with Categories and links to products pages
Products are filtered by Category.
Global helper function added in ApplicationController for categories

Commit:
Styled sign up and log in page

Commit:
Added alert messages to main application layout

Commit:
Commented out not needed dropdown menu in main navigation

o2badad committed:
Look into why db/schema file was modified after pull.

o2badad committed:
Merge branch 'MurrietaRoR/master' into develope

o2badad committed:
db/schema file updated after db:migrate rake command.
This was perfmormed after a merge with MurrietaRoR/master

o2badad committed:
Merge pull request #9 from o2badad/develope

Categories and Products listed dynamically.

Commit:
Moved stylesheets inside "base" folder for automatic import using wildcard

o2badad commented on Gemfile:
```diff
@@ -3,6 +3,7 @@ ruby '2.2.2'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '4.2.3'
+
```
What were the changes you made? I do not see anything but a new line in the Gemfile.

o2badad committed:
Rewrote prducts listing of selected categories.
Use partials to render each product. Category show method collects its products
to render for view.

o2badad committed:
Merge branch 'master' into MurrietaRoR/master
